docs/src/pages/components/link/react.mdx

```diff
@@ -80,74 +106,68 @@ The Link component primarily uses four css pseudo classes. These are `:active`,
 
 To override styling on all Links, you can set the Amplify CSS variables or use the built in `.amplify-link` class.
 
-```css
-/* styles.css */
-:root {
-  --amplify-components-link-color: red;
-}
-/* OR */
-.amplify-link {
-  color: red;
-}
-```
-
-To replace Link styling, unset it:
-
-```css
-.amplify-link {
-  all: unset;
-  /* Add your styling here */
-}
-```
+<Example>
+  <LinkGlobalStylingExample />
+  <ExampleCode>
+    ```css
+    /* styles.css */
+    :root {
```

Comment:
I don't think `:root` works for overriding css variables because of how we do out theme, global styling rule should be `[data-amplify-theme]`